src/SFA.DAS.EmployerIncentives.Api/appsettings.acceptancetests.json

```diff
@@ -0,0 +1,9 @@
+{
+  "SFA.DAS.EmployerIncentives.OuterApi": {
```

Comment:
Shouldnt need this file - any configuration you require for acceptance tests should be done using the InMemoryCollection

Reply:
It's to override the AzureStorage Configs. Our acceptance tests are not going to call the External APIs.   We are going to use wiremock to test the API calls, but we need to override the config values